mods/ts/weapons/energyweapons.yaml

```diff
@@ -40,29 +43,21 @@ LtRail:
 		DamageTypes: Prone100Percent, TriggerProne, ExplosionDeath
 
 MechRailgun:
-	ReloadDelay: 60
+	Inherits: ^Railgun
 	Range: 8c0
 	Burst: 2 # for alternating muzzle offsets, dmg/s identical to original
 	BurstDelay: 60
 	Report: railuse5.aud
 	Projectile: AreaBeam
-		Speed: 20c0
-		Duration: 3
-		DamageInterval: 2
-		Width: 80
-		BeyondTargetRange: 0c64
-		Blockable: true
 		Color: 00FFFFC8
 	Warhead@1Dam: SpreadDamage
-		Spread: 42
```

Comment:
>This is another good candidate for TargetDamageWarhead once that supports more projectiles.

Is there a way to "tag" such information, so that if `TargetDamageWarhead` reaches this level a reminder appears? Else I have the feeling that after a merge such notes are lost? Creating an extra issue seems to be an overkill.